From 385972ded7c35ac2e7e4a3e396b4387df40add2e Mon Sep 17 00:00:00 2001
From: Matthias Kretz <m.kretz@gsi.de>
Date: Thu, 2 Feb 2023 10:48:10 +0100
Subject: [PATCH 483/547] libstdc++: Document timeout and timeout-factor of
 simd tests

Signed-off-by: Matthias Kretz <m.kretz@gsi.de>

libstdc++-v3/ChangeLog:

	* testsuite/experimental/simd/README.md: Document the timeout
	and timeout-factor directives. Minor typo fixed.

(cherry picked from commit b0f4b166ada3b92da5f2917ac3f4397e99d1b58f)
---
 libstdc++-v3/testsuite/experimental/simd/README.md | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/libstdc++-v3/testsuite/experimental/simd/README.md b/libstdc++-v3/testsuite/experimental/simd/README.md
index b82453df403..ef8b7c33de7 100644
--- a/libstdc++-v3/testsuite/experimental/simd/README.md
+++ b/libstdc++-v3/testsuite/experimental/simd/README.md
@@ -139,7 +139,13 @@ allowed_distance)` macros.
   test then shows as "XFAIL: ...". If the test passes, the test shows "XPASS: 
   ...".
 
-All patterns are matched via
+* `timeout: <number>`
+  Set the timeout of this test to `<number>` seconds.
+
+* `timeout-factor: <number>`
+  Multiply the default timeout with `<number>`.
+
+All patterns except `timeout` and `timeout-factor` are matched via
 ```sh
 case '<test context>' in
   <pattern>)
@@ -147,7 +153,7 @@ case '<test context>' in
   ;;
 esac
 ```
-The `<CXXFLAGS pattern>` is implicitly adds a `*` wildcard before and after the 
+The `<CXXFLAGS pattern>` implicitly adds a `*` wildcard before and after the 
 pattern. Thus, the `CXXFLAGS` pattern matches a substring and all other 
 patterns require a full match.
 
-- 
2.25.1

